app/routes.py

```diff
@@ -1,2 +1,239 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response,request
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from datetime import datetime 
+#what comes from the "from" is the package name
+#what comes after the import is a variable that matches the name of thing we want to import
+tasks_bp = Blueprint("tasks", __name__,url_prefix=("/tasks"))
+
+goals_bp = Blueprint("goals", __name__, url_prefix=("/goals"))
+
+@goals_bp.route("",methods=["POST", "GET"])
+def handle_goal():
+      if request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body:
+            return{
+                "details": "Invalid data"
+            },400
+
+        new_goal = Goal (
+            title=request_body["title"]
+        )
+
+        db.session.add(new_goal)
+        db.session.commit()
+
+        return {
+            "goal": {
+                "id":new_goal.goal_id,
+                "title":new_goal.title
+            }
+        }, 201
+        
+      elif request.method == "GET":
+        sorting_goals= request.args.get('sort') 
+        list = None
```

Comment:
`list` will definitely have a value by the time the following conditions have been checked (each assigns to list), so the `None` here isn't all that dangerous, but it's a little confusing. If we're trying to fill in a list with deferred lookups, it's usually better to initialize the variable with an empty list rather than `None`, as it works as expected with iteration, while `None` would crash.

Also, consider using a name other than `list`, since that hides the class name `list`. Python lets us do this, but again, it can be confusing.